workflows/src/test/java/com/r3/developers/serializers/MemberX500NameSerializer.java

```diff
@@ -0,0 +1,15 @@
+package com.r3.developers.serializers;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import net.corda.v5.base.types.MemberX500Name;
+
+import java.io.IOException;
+
+public class MemberX500NameSerializer extends JsonSerializer<MemberX500Name> {
+    @Override
+    public void serialize(MemberX500Name value, JsonGenerator generator, SerializerProvider serializers) throws IOException {
+        generator.writeString(value.toString());
```

Comment:
where is the serialized output stored ?

Reply:
We're overriding a function from Jackson > JsonSerializer which handles the generator and the value it produces internally